pkg/vsphere/disk/vmdk.go

```diff
@@ -58,7 +64,34 @@ func LockedVMDKFilter(vm *mo.VirtualMachine) bool {
 	return vm.Runtime.PowerState == types.VirtualMachinePowerStatePoweredOn
 }
 
-func IsLockedError(err error) bool {
-	// TODO: add check here for actual error
-	return true
+func IsLockedError(op trace.Operation, err error) bool {
```

Comment:
some client would want to know which disk is locked, and this method already go through all the messages, it would be nice to return the locked disk name as well.